CMakeLists.txt

```diff
@@ -27,6 +27,19 @@ else()
   add_library(skity SHARED)
 endif()
 
+if(ANDROID)
+  find_library(
+          gles-lib
+          GLESv3
+  )
+  find_library(
```

Comment:
The `proc_loader` property in GPUContext is used to load required GL function at runtime. So during compile time, skity does not need to link gles or egl library. I have a seperate android example project to show this usage, and I hope skity not link libEGL.so or libGLESv3.so at compile time.

Reply:
OK, i will fix it

Reply:
https://github.com/RuiwenTang/Skity-Android

Reply:
ok , i will